app/routes/first-time/eligibility/claim/car-details.js

```diff
@@ -14,9 +18,40 @@ module.exports = function (router) {
     })
   })
 
-  // TODO: Add form validation.
   router.post('/first-time-claim/eligibility/:reference/claim/:claimId/car', function (req, res) {
     UrlPathValidator(req.params)
-    return res.redirect(expenseUrlRouter.getRedirectUrl(req))
+
+    console.log(req.body)
+
+    var carExpense = new CarExpense(
+      req.params.claimId,
+      req.body.from,
+      req.body.to
+    )
+
+    var tollExpense
+    if (req.body.toll) {
+      tollExpense = new TollExpense(
+        req.params.claimId,
+        req.body[ 'toll-cost' ],
+        req.body.from,
+        req.body.to
+      )
+    }
+
+    var parkingExpense
+    if (req.body[ 'parking-charge' ]) {
+      parkingExpense = new ParkingExpense(
+        req.params.claimId,
+        req.body[ 'parking-charge-cost' ],
+        req.body.from,
+        req.body.to
+      )
+    }
```

Comment:
the way you are constructing these means if a top one throws a validation error the others won't get called, should supply in the req.body.toll/ parking-charge to constructor of carExpense and validate it there.
Persisting a single carExpense with insertCarExpenses will insert the three expenses